### ISSUE TITLE
Add SafariServices dependency to the documentation

### ISSUE THREAD

**Issue by guthyerrz** [AppLovin/iOS-SDK-Demo#23]
In your documentation the SafariServices framework dependency is not mentioned!

**Comment by thomasmso** [comment]
Thank you for bringing this up - it is already in our UI team's pipeline to push the documentation update, but there's no ETA for that.

Also, how are you integrating AppLovin's SDK? Via the first class framework (`AppLovinSDK.framework`) or as a raw binary + headers (`libAppLovinSdk.a`)?

**Comment by guthyerrz** [comment]
I'm integrating using the .unitypackage

**Comment by thomasmso** [comment]
Gotcha - so that's via the raw binary + headers which makes sense the dependency wasn't linked automatically.  We'll update the documentation accordingly, and remove the hard-linking of the dependency in the next release. Thank you!

**Comment by IvyKun** [comment]
I want to mention that my app was rejected by apple for using the SafariServices framework. Because it mimics the safari functionality (words from apple, not me).

I had to go back to a previous version of applovin (5.0.2) that don't use this.

I will not be updating applovin sdk while they use SafariServices.

**Comment by thomasmso** [comment]
@IvyKun - Any more info that you can provide? SafariServices is supposed to to offer Safari-like functionality to developers, it is surprising that Apple used that as a point of rejection.

**Comment by IvyKun** [comment]
I was publishing just an update of my game with new sdks of my advertising networks and a few fixes. Apple rejected it with this message:

> Performance - 2.3.1 
> Your app contains hidden features
> 
> Design - 4.2 
> Your app provides a limited user experience as it is not sufficiently different from a mobile browsing experience. As such, the experience it provides is similar to the general experience of using Safari.

Goes without saying the game is a completely normal game, nothing even close to a browser experience. So of course, my first reaction was just answer with a rant saying that it didn't even make sense saying a game is even similar to a browser. 
Next day I remembered the SafariServices framework and I answered Apple again saying that I had that framework and probably that caused that rejection. Not sure if it was a bot, or a reviewer just getting lost in a browser after an ad or something like that.

I uploaded a new version with the old Applovin SDK and without SafariServices and it get approved. But I didn't get a message back from Apple confirming that it was because of SafariServices or anything really, so I'm not 100% sure what happened.
